### DISCUSSION TITLE
Can't open the program even when installed.

### DISCUSSION THREAD

**Discussion by Titaniumbot360** [ggarra13/mrv2#73] [Q&A]
Hello there. So for some reason, after I installed the program, when I double clicked to install, since I use the pre-built/exe version, the program won't open for some reason.

Is there any sort of fix for it? Or maybe there were some programs/dependencies that I haven't installed for it to properly open, so, any help will be appreciated.

**Comment by ggarra13** [comment]
It isn't clear to me what's happening to you.   Did you run the installer (where it asks you a location to install it, optionally add it to the PATH and create a shortcut on the Desktop)?

If you cannot run the installer, it may be that you are trying to run it from the browser and Chrome/Windows protects you about it.  See Notes on Installation for Windows on the main code page at https://github.com/ggarra13/mrv2

If you did run the installler, the easiest way to run mrv2 it is to create a shortcut on the Desktop.  The installer has a tick option for  it while you install it (you might have missed it while installing).

**Reply by Titaniumbot360** [reply]
I did checked to have the installer create the shortcut, and no I did not let the installer add the PATH to it.

**Comment by ggarra13** [comment]
Ok.  Some silly questions first (I apologyze if they are too obvious). 

1) Did it create a shortcut in your Desktop?  It should look like an eye and be named mrv2-64 v0.6.1.
2) Did you check with Windows Explorer (the Folder icon on the taskbar) that it actually installed it in the location you specified when running the installer (by default C:\Program Files\mrv2 0.6.1)?

**Reply by Titaniumbot360** [reply]
The shortcut did appear in my Desktop and I did install the program in the default install location

**Comment by ggarra13** [comment]
Ok.  If you click on the shortcut and it does not start up, the likely case is that one of the preferences file got corrupted (you might have had a previous version perhaps).   Check the directory in your home directory (C;/Users/Your-Login/.filmaura/mrv2.prefs) and get rid of that file (rename it and it should get recreated on start-up.

**Comment by ggarra13** [comment]
Also, something obvious.  You must be running Windows 8.1 at least to run mrv2.  If you have an earlier version, it won't work.

**Reply by Titaniumbot360** [reply]
No because I had Windows 11 running on my computer where I installed the program.

**Comment by ggarra13** [comment]
You don't give me much to work with.  Please let me know what you tried and what happened (ie. do you have a mrv2.prefs file?).

1) Do the following and please report the output you get (if any):

- Start a Windows console by going to the Search item in the taskbar of Windows 11.  Type cmd.exe.
- It should start the Windows console.
- Type:
    `cd "C:\Program Files\mrv2 0.6.1\bin"`
- Type:
`mrv2.exe`
- Please report here the output you get.

2) Let me know what language your Windows is.   That is, are you running it in English, Spanish or some other language.

**Reply by Titaniumbot360** [reply]
I tried it and...still no response, which is odd considering that, I had the program installed on my laptop months ago and it worked well. Either it could be the antivirus or something else. Also, I couldn't find the mrv2 prefs file, and my OS' language is in English.

![image](https://github.com/ggarra13/mrv2/assets/61374997/3a31da0c-f874-43fb-ad8f-b691fbfd6eb6)

**Reply by Titaniumbot360** [reply]
wait, nevermind, I found my prefs file and this is what it says.

![image](https://github.com/ggarra13/mrv2/assets/61374997/30b0243a-0012-4c14-b64a-80c0d48ca1f3)

**Comment by ggarra13** [comment]
Yes, that is a corrupt mrv2.prefs file.  Get rid of it and start the program again.

**Reply by Titaniumbot360** [reply]
I did delete the pref file and opened the program. Still can't open.

**Comment by ggarra13** [comment]
Ok.  I am about to give up.  I am thinking either your antivirus is preventing it from running or you have something broken on your system.  However, a final check.  Check these envrionment variables in Settings->About->Advanced system settings->Environment Variables.

Check in both the user and system variables the values of:

1. LANGUAGE            - it should be empty
2. LC_ALL                   - it should be empty
3. OCIO                     - it should be empty or point to a valid OCIO2 config file
4. PYTHONPATH       - it should point to a python3.10 installation or be empty
5. PATH                     - check for conflicting or non-existing paths

**Reply by Titaniumbot360** [reply]
Actually, on second thought, I think there is something broken on my system, but I really don't know on what it is.

**Reply by Titaniumbot360** [reply]
Also, regarding variables...

1. I don't think there is a language variable, nor does lc_all and ocio on any of my pc's variables
3. there is pythonpath on my user variables, but it is set in the path label
4. the path on my system variables is set to Java.

**Comment by ggarra13** [comment]
I am sorry that I couldn't be of more help.

**Reply by Titaniumbot360** [reply]
it's okay.

**Reply by Titaniumbot360** [reply]
I guess the only solution to fix it is that, if you plan to update the software, maybe update the installer so that it also includes some additional components for it to fix my situation whenever the update comes.

**Reply by Titaniumbot360** [reply]
And also so that other users can be prevented from encounter the same issues as I have.

**Comment by Titaniumbot360** [comment]
Hello again. Figured I share an update regarding my problem, which is that I'm beginning to assume that it may be a GPU problem as I have an Nvidia GTX while my laptop has an Intel UHD. I think it may be related to that.

**Comment by ggarra13** [comment]
Sorry to have left you alone.  I doubt the graphics card is the issue, as I also have a GTX card.  Download this debug version of mrv2.exe and replace (make a back-up first) of the mrv2.exe in the bin/ installation directory of mrv2 v0.6.4.

https://mega.nz/file/KeZCBBLR#S90kq38mb6C48Z-HgZMUZIL80dSrFIXzH04LxB2-MkA

When you run this new debug version, it should open a command console and print out some debug info.  If it closes right away, please try to capture a video of it as it may still have printed out some stuff that may give me a hint of what's wrong.

Also, if it closes right away, from a command console, in the bin\ directory of mrv2 v0.6.4 try running:

mrv2.exe -h

and see if it prints the command-line help information.

**Reply by Titaniumbot360** [reply]
Okay, I run the debug and this error popped up. Must be a missing dll file.
![Screenshot 2023-08-05 025328](https://github.com/ggarra13/mrv2/assets/61374997/6b767cf9-e679-4ffc-9436-aa6e7a0f5292)

**Comment by ggarra13** [comment]
Sorry.  I should have given you the full installer, instead of just the executable, as the debug libs change names.  Here's the full installer.  Let me know how it goes.

https://mega.nz/file/PKYDXZSa#I9QvsOT65cihWkVf-EiUJvDrBBFb3kB-sU129k6LCoo

**Reply by Titaniumbot360** [reply]
Installer works, and when I open the app, this error pops up.
![image](https://github.com/ggarra13/mrv2/assets/61374997/557a198a-2b99-420d-91d9-4d0a1a3359b9)

**Comment by ggarra13** [comment]
That's a strange error.  I think you may have an older Microsoft C++ Redistributable and that may be causing the problem.  Please try downloading and installling the latest version of MSVC runtimes from:

https://learn.microsoft.com/en-us/cpp/windows/latest-supported-vc-redist?view=msvc-170

Specifically this file:

https://aka.ms/vs/17/release/vc_redist.x64.exe

**Comment by ggarra13** [comment]
After you install the distributable, try installing and running the Release version of mrv2.  The debug version I gave you on these pages may or may not work.
